.github/workflows/tests.yml

```diff
@@ -73,3 +132,48 @@ jobs:
         run: bats test
         env:
           GITHUB_API_TOKEN: ${{ github.token }}
+
+  documentation-site:
+    needs: changes
+    # only run if
+    # - changes to documentation
+    # - pull_request (workflows/docs.yml deploys on main branch)
+    if: ${{ github.event_name == 'pull_request' && needs.changes.outputs.documentation == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # fetch all commits to get git log info for Vuepress
+          fetch-depth: 0
+
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            docs:
+              - 'docs/**'
```

Comment:
Where is the output from this step used?

Reply:
Ah, yes, that was from prior testing of the filter action. I will remove.